### PR TITLE
Release/1.1.0

### PR DESCRIPTION
**Pull request by agger-magenta**

### CODE REVIEW

File: NEWS.rst [hunk 1]
```diff
@@ -1,3 +1,29 @@
...
+* Searching for/filtering on Boolean attributes fixed.
+* Proof-of-concept infrastructure for Gitlab Runners added.
+* New dummy SQL fixtures for integration with the MO test suite.
+
+Documentation improved:
+
+* API reference documentation vastly improved.
+* API tutorial updated and made more user friendly.
+* Behaviour of wildcard searches properly documented.
+* Proper documentation of settings.py.
```
Comment (danchr) [inline NEWS.rst]: You monospace for path names:
```suggestion
* Proper documentation of ``settings.py``.
```

File: NEWS.rst [hunk 2]
```diff
@@ -1,3 +1,29 @@
+Version 1.1.0, 26 March 2019
+----------------------------
+
+New features:
+
+* Python dependencies updated.
+* Delay import of DB_STRUCTURE file until after configuration file is
```
Comment (danchr) [inline NEWS.rst]: Use monospace for variables and constants:

```suggestion
* Delay import of ``DB_STRUCTURE`` file until after configuration file is
```

File: NEWS.rst [hunk 3]
```diff
@@ -1,3 +1,29 @@
...
+  read.
+* Improve speed and configurability of test databases. Disable fsync for
+  speed and allow Jenkins to use long branch names without failing.
+* Fix calculation of BASE_DIR in tests.
```
Comment (danchr) [inline NEWS.rst]:
```suggestion
* Fix calculation of ``BASE_DIR`` in tests.
```

File: NEWS.rst [hunk 4]
```diff
@@ -1,3 +1,29 @@
...
+* Make API endpoints trailing slash agnostic. 
+* Clear caches in test code when patching DB structures.
+* Validation of objects with additional attributes simplified.
+* Searching for/filtering on Boolean attributes fixed.
```
Comment (danchr) [inline NEWS.rst]: Not a proper name, and this is shorter:
```suggestion
* Fixed filtering on boolean attributes.
```

File: NEWS.rst [hunk 5]
```diff
@@ -1,3 +1,29 @@
...
+
+* Python dependencies updated.
+* Delay import of DB_STRUCTURE file until after configuration file is
+  read.
+* Improve speed and configurability of test databases. Disable fsync for
+  speed and allow Jenkins to use long branch names without failing.
+* Fix calculation of BASE_DIR in tests.
+* Make API endpoints trailing slash agnostic. 
+* Clear caches in test code when patching DB structures.
+* Validation of objects with additional attributes simplified.
```
Comment (danchr) [inline NEWS.rst]: Odd word order:
```suggestion
* Simplified validation of objects with additional attributes.
```

File: NEWS.rst [hunk 6]
```diff
@@ -1,3 +1,29 @@
+Version 1.1.0, 26 March 2019
+----------------------------
+
+New features:
+
+* Python dependencies updated.
+* Delay import of DB_STRUCTURE file until after configuration file is
+  read.
+* Improve speed and configurability of test databases. Disable fsync for
+  speed and allow Jenkins to use long branch names without failing.
```
Comment (danchr) [inline NEWS.rst]: Not specific to Jenkins:
```suggestion
  speed and allow using long directory names without failing.
```

File: NEWS.rst [hunk 7]
```diff
@@ -1,3 +1,29 @@
+Version 1.1.0, 26 March 2019
+----------------------------
+
+New features:
+
+* Python dependencies updated.
+* Delay import of DB_STRUCTURE file until after configuration file is
+  read.
+* Improve speed and configurability of test databases. Disable fsync for
```
Comment (danchr) [inline NEWS.rst]:
```suggestion
* Improve speed and configurability of test databases. Disable ``fsync`` for
```

File: NEWS.rst (hunk 4 above)
Comment (agger-magenta) [reply NEWS.rst]: "boolean" is wrong.

Comment (danchr) [reply NEWS.rst]: > "boolean" is wrong.

Indeed it is, [apparently](https://english.stackexchange.com/a/13760)…

**Pull request merged**